src/component/button/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';

```

Comment:
I suggest we implement the [atomic design](https://atomicdesign.bradfrost.com/chapter-2/).
We can move this file into `src/components/atoms/Button/index.tsx`